ui/ts/pages/cluster.ts

```diff
@@ -158,16 +158,15 @@ module AdminViews {
               Metrics.Select.Avg(_sysMetric("rss"))
                 .title("Memory")
             ).format(Utils.Format.Bytes).title("Memory Usage")
-              .tooltip("The memory in use across all nodes.")
-
+              .tooltip("The average memory in use across all nodes.")
```

Comment:
I think I just changed that in my PR. It's definitely not average, it's the total memory used.